src/modules/leaderboard.js

```diff
@@ -0,0 +1,74 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/PYiYgQE33xPmsyutxO0U/scores';
+
+const listScores = () => document.querySelector('#list-scores');
+
+const checkEmpty = () => {
+  if (listScores().innerHTML === '') {
+    const listItem = document.createElement('li');
+    listItem.textContent = 'Recorded No Scores';
+    listScores().appendChild(listItem);
+  }
+};
+
+const scoresDisplay = (scores) => {
+  scores.forEach((scoreObject) => {
+    const listItem = document.createElement('li');
+    listItem.textContent = `${scoreObject.user}: ${scoreObject.score}`;
+    listScores().appendChild(listItem);
+  });
+};
+
+const getScoreObjects = async () => {
+  const response = await fetch(url);
+  const scores = (await response.json()).result;
+
+  scoresDisplay(scores);
+};
+
+const refresh = async () => {
+  listScores().innerHTML = '';
+  await getScoreObjects();
+  checkEmpty();
+};
+
+const addRefresh = () => {
+  const button = document.querySelector('#refresh');
+  button.addEventListener('click', () => {
+    refresh();
+  });
+};
+
+window.addEventListener('load', () => {
+  refresh();
+});
+
+addRefresh();
+
+const postScore = (scoreObject) => {
+  fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify(scoreObject),
+  });
+};
+
+const submitAddFunc = () => {
```

Comment:
- [ ] KIndly check on your form method, when I click submit I get the error below in my browser:
![image](https://user-images.githubusercontent.com/61408860/186678127-5f2328b8-a243-4153-ab20-fbb1c6a9f659.png)

Reply:
Hello @danmainah, I have checked in my browser and it works fine

![image](https://user-images.githubusercontent.com/39293404/186681827-a572202f-46e1-4c1a-b0ee-eaaa32cf4654.png).

I have added AAA 123 now

Reply:
- [ ] As stated by the previous reviewer, as the same problem exists and I stated in [this loom video] (https://www.loom.com/share/e9514ecd04874789ac1defd72de57b27) why this happens if you make sure that your project work on your side kindly make a loom video to demonstrate it is working and the next reviewer will approve it.